omero/developers/python3-migration.rst

```diff
@@ -0,0 +1,108 @@
+Migration from OMERO 5.5 (Python 2) to OMERO 5.6 (Python 3)
+===========================================================
+
+This page serves as a collection of tips the OME team used to
+get through the migration to Python 3. This is not a complete
+guide but may serve as a useful starting point.
+
+For more information, please see a dedicated Python 3 page like
+http://python-future.org/.
+
+Futurize
+--------
+
+Installing `future` from Python 3 is now required for all OMERO
+Python components. This library comes with a tool, `futurize`
+which will perform many of the basic transformations needed to
+migrate Python 2 code to Python 3:
+
+::
+
+    futurize -0 your_file.py
+
+Add `-w` to update the file in place.
+
+`print()`
+^^^^^^^^^
+
+The most common transformation needed is adding parentheses around
+print statements since print is no longer a keyword.
+
+`dict.keys()`
+^^^^^^^^^^^^^
+
+The return value from the `keys()` method of dictionaries is of type
+`dict_keys` and no longer has methods like `sort()`. Wrap with a call
+to list if you need the previous behavior: `list(my_dict.keys())`.
+
+
+Strings
+-------
+
+One of if not the major hurdle in upgrading from Python 2 to Python 3 is the shift in
```

Comment:
```suggestion

```

Reply:
Can you explain why you suggest removing this?

Reply:
It is a very long sentence which has not much informative value. Makes tedious to read- if this is not the major hurdle, then why do I need to know that that is not the major hurdle ? Obviously it is one of the hurdles, I knew that anyway...

Reply:
It's a pretty common idiom, but I'm pretty sure we can just say it is the major hurdle. In other words: "this is going to be the problem you run into."

Reply:
New attempt pushed.

Reply:
Read the new commit about strings and like it. Ta